<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1421</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Inspection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_screen_cam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 218, 218);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pylonopen">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>basler open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setgt">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set gt</string>
    </property>
   </widget>
   <widget class="Canvas" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 203, 203);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addgt">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add box</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_gts">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>420</y>
      <width>91</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delgt">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>390</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>del box</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_label_classes">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>81</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>358</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_continuous">
       <property name="text">
        <string>continuous</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_hwtrigger">
       <property name="text">
        <string>hwtrigger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_swtrigger">
       <property name="text">
        <string>swtrigger</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_test">
       <property name="text">
        <string>test</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_savegt">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>690</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_loadgt">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>720</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_savegtpath">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>690</y>
      <width>28</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_loadgtpath">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>720</y>
      <width>28</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_savegtpath">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>690</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_loadgtpath">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>720</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pjname">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>660</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set_trigger">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>100</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set_trigger</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>acqutime</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>triggerdelay</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_triggerdelay"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_acqutime"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_infer_iou">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_infer_conf">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.450000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_inference">
        <property name="text">
         <string>inference</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>IOU</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>conf</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>infernece</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_insp_iou">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>96</width>
       <height>24</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_swtrigger">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>sw_trigger</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_fiducial">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>390</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>fiducial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_webcamopen">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>c920 open</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1421</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
